rotkehlchen/assets/converters.py

```diff
@@ -514,6 +517,16 @@ def asset_from_poloniex(poloniex_name: str) -> Asset:
     return Asset(our_name)
 
 
+def asset_from_bitstamp(bitstamp_name: str) -> Asset:
+    if not isinstance(bitstamp_name, str):
+        raise DeserializationError(f'Got non-string type {type(bitstamp_name)} for bitstamp asset')
+
+    if bitstamp_name in UNSUPPORTED_BITSTAMP_ASSETS:
```

Comment:
Remove this check and `UNSUPPORTED_BITSTAMP_ASSETS` if there are none. Why add a new variable and an if check for a no-op?

Reply:
I've done it for giving consistency with the existing code and for having an implementation ready if they ever add an asset we won't support :S. I understand why removing it, no problem.